eth/downloader/skeleton.go

```diff
@@ -582,6 +596,14 @@ func (s *skeleton) processNewHead(head *types.Header, force bool) bool {
 
 	lastchain := s.progress.Subchains[0]
 	if lastchain.Tail >= number {
+		// If the chain is down to a single beacon header, and it is re-announced
+		// once more, ignore it instead of tearing down sync for a noop.
+		if lastchain.Head == lastchain.Tail {
```

Comment:
It's unclear to me.. 

So we have the lastchain whose head is equal to tail(it means we haven't fetched any skeleton header yet in this cycle) and we get a lower point announcement but the corresponding skeleton is already existent. 

Any particular reason for specialize this scenario?